api/openapi.yaml

```diff
@@ -2407,6 +2769,183 @@ components:
               Rootkit: '#/components/schemas/RootkitFindingInfo'
               Exploit: '#/components/schemas/ExploitFindingInfo'
 
+    ScanEstimations:
+      type: object
+      properties:
+        count:
+          description: Total ScanEstimations count according to the given filters
+          type: integer
+        items:
+          description: List of ScanEstimations according to the given filters
+          type: array
+          items:
+            $ref: '#/components/schemas/ScanEstimation'
+
+    ScanEstimation:
+      type: object
+      properties:
+        id:
+          type: string
+        assetIDs:
+          description: List of asset IDs to be estimated
+          type: array
+          items:
+            type: string
+        state:
+          $ref: '#/components/schemas/ScanEstimationState'
+        startTime:
+          type: string
+          format: date-time
+        endTime:
+          type: string
+          format: date-time
+        assetScanEstimations:
+          description: AssetScanEstimations which make up this ScanEstimation
+          type: array
+          items:
+            $ref: '#/components/schemas/AssetScanEstimationRelationship'
+        scanTemplate:
+          $ref: '#/components/schemas/ScanTemplate'
+        summary:
+          $ref: '#/components/schemas/ScanEstimationSummary'
+        revision:
+          type: integer
+
+    ## TODO fill out the rest of the fields for the ScanEstimationRelationship.
```

Comment:
what is the impact of not having the rest filled?

Reply:
If you do `GET assetScanEstimation/<uuid>?$expand=ScanEstimation` then you will only get the ScanEstimation fields specified here and nothing else.